content/integrations/AWS-Integration/AWS-CloudFormation-Installation.md

```diff
@@ -12,31 +12,22 @@ This installation method leverages a CloudFormation script which can be viewed [
 
 After you make a new AWS integration in CloudWisdom, the script populates a read-only IAM role in your AWS account and links it using the integration’s Account ID and External ID. Once created, it may take a few minutes for the status to be updated.
 
-## Create an AWS Integration in CloudWisdom
-1. Open CloudWisdom and navigate to Integrations > AWS.  
-2. Create a new AWS integration. Ensure that the below tabled properties are set according to your needs.
+## 1. Create an AWS Integration in CloudWisdom
+1. Open CloudWisdom and navigate to **Integrations** > **Amazon Web Services**.  
+2. Select **+ Add Integration**.
```

Comment:
If you go Integrations -> AWS, you don't have this step

Reply:
If they are using this guide to manually create an IAM role then they likely have already made their first one via the onboarding wizard, right?

Reply:
most probably yes, but exceptions are possible